_data/projects/CASE.yml

```diff
@@ -0,0 +1,14 @@
+name: CASE
+desc: A frameworkless and Open Source tool to build CRUD applications as dashboards or admin panel.
+site: https://case.app
+tags:
+  - appbuilder
+  - typescript
+  - adminpanel
+  - CRUD
+  - frameworkless
+  - opensource
+  -lowcode
+upforgrabs:
+  name: CASE
+  link: https://github.com/casejs/CASE/issues
```

Comment:
This `CASE` value should be a label in your project which is used to tag issues that are ideal for new contributors, and the URL should be a link to that list of issues e.g. `https://github.com/casejs/CASE/labels/help%20wanted`